pkg/keyspace/keyspace.go

```diff
@@ -54,6 +56,20 @@ func GetKeyspaceNameBySettings() (keyspaceName string) {
 	return keyspaceName
 }
 
```

Comment:
```suggestion
var (
    KeyspaceID = GetKeyspaceIDBySettings()
)
```

Reply:
In `GetKeyspaceIDBySettings`, there is a call to `config.GetGlobalKeyspaceName`. If handled directly in this way, the obtained value should be an uninitialized one.